testing/kokoro/system_tests.sh

```diff
@@ -141,4 +141,36 @@ date
 OUTFILE=gotest.out
 2>&1 go test -timeout $TIMEOUT -v . $TARGET | tee $OUTFILE
 
+set +x
```

Comment:
I would be careful about setting this, because if any secrets are passed as arguments they will show up in public logs.

Reply:
`set -x` turns logging on, `set +x` turns it off. But, whoops, I meant to turn off `set -e` (set at the top of the file) so a test failure wouldn't stop execution. Fixed.